AegeanTools/AeRes.py

```diff
@@ -130,15 +131,15 @@ def make_model(sources, shape, wcshelper, mask=False, frac=None, sigma=4):
         if not np.all(np.isfinite([ymin, ymax, xmin, xmax])):
             continue
 
-        if logging.getLogger().isEnabledFor(logging.DEBUG):  # pragma: no cover
-            logging.debug("Source ({0},{1})".format(src.island, src.source))
-            logging.debug(" xo, yo: {0} {1}".format(xo, yo))
-            logging.debug(" sx, sy: {0} {1}".format(sx, sy))
-            logging.debug(" theta, phi: {0} {1}".format(theta, phi))
-            logging.debug(" xoff, yoff: {0} {1}".format(xoff, yoff))
-            logging.debug(" xmin, xmax, ymin, ymax: {0}:{1} {2}:{3}".format(
+        if logging.getLogger().isEnabledFor(logger.debug):  # pragma: no cover
```

Comment:
Should be
`if logger.isEnabledFor(logging.DEBUG)`

Reply:
Blah. Yes.